src/Components/CommentList.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Comment from './Comment';
+
+const CommentList = ({ comment }) => (
+  <div>
+    {' '}
+COMMENTS
+    {comment
+      .map(curCom => <Comment curCom={curCom} key={curCom.id} />)}
```

Comment:
```suggestion
      .map(singleComment => <Comment commentData={singleComment} key={singleComment.id} />)}
```
Не сокращай названия переменных

Reply:
+